ios/CameraView+AVCaptureSession.swift

```diff
@@ -203,6 +233,30 @@ extension CameraView {
         }
         device.activeColorSpace = avColorSpace
       }
+      if #available(iOS 13.0, *) {
+        if enableDepthData?.boolValue == true && !device.activeFormat.supportedDepthDataFormats.isEmpty {
+          if let depthDataFormat = depthDataFormat as String? {
+            // Check if the selected depth format is supported as part of the activeFormat
+            let avCaptureDepthDataFormat = depthDataFormat
+            if device.activeFormat.supportedDepthDataFormats.map({
+              CMFormatDescriptionGetMediaSubType($0.formatDescription).toString()
+            }).contains(avCaptureDepthDataFormat) {
```

Comment:
instead of two loops you could probably just create a single `find(where:)` loop?